test/foods-test.js

```diff
@@ -0,0 +1,71 @@
+var assert    = require('chai').assert;
+var webdriver = require('selenium-webdriver');
+var until = webdriver.until;
+var test      = require('selenium-webdriver/testing');
+var frontEndLocation = "http://localhost:8080"
+var Food = require('../lib/food')
+var pry = require('pryjs')
+var $ = require('jquery');
+var host = require('../lib/config').host
+
+test.describe('testing food index', function() {
+  var driver;
+  this.timeout(1000000);
+
+  test.beforeEach(function() {
+    driver = new webdriver.Builder()
+      .forBrowser('chrome')
+      .build();
+  })
+
+  test.afterEach(function() {
+    driver.quit();
+  })
+
+  test.it("lists all the foods on load", function() {
+    // var foodCount = Food.getAllFoods().then(function(data){
```

Comment:
@jk1dd this is interesting. where is the error coming from? that seems weird that it would work in one situation and not another. is there something else you're doing differently?